keymaps/ink.cson

```diff
@@ -14,12 +14,20 @@
 'ink-console':
   'shift-space': 'unset!'
 
+'.platform-win32 .ink, .platform-linux .ink':
+  'ctrl-c': 'native!'
+
+'.platform-darwin .ink':
+  'cmd-c': 'native!'
+
 'atom-text-editor:not([mini])':
   'escape': 'inline:clear-current'
 
 '.platform-darwin atom-text-editor:not([mini])':
   'cmd-i cmd-c': 'inline-results:clear-all'
+  'cmd-t': 'inline-results:toggle'
```

Comment:
This opens the file switcher on mac. alt-t is actually not taken (aside from inserting a special character) so that may actually be better.